### PR TITLE
feature/strategy_order_description

### PR DESCRIPTION
**Pull request by miguelromero2000**
- Added the possibility to set a description message in all types of orders.
- Added the possibility to show that description message with TradingViewPlugin report for each operation in TradingView. This is extremely useful in order to debug all orders with TradingView and to know why one order was executed just looking at the description label.
- Improved supertrend indicator which now will return more info, useful to know the trend and performing any necessary action. The Supertrend tuple will now return: ['trend', 'changed', 'switched_to', 'current_trend']
- Example about how to use the new description field in any Strategy:
        self.buy = qty, entry
        self.stop_loss = qty, stop
        self.description = 'RSI70 passed\nMFI65 passed'

### CODE REVIEW

**Comment by miguelromero2000** [comment]
Example of how it looks like in TradingView.
Now debugging orders is awesome and easy :)
image

**Comment by saleh-mir** [comment]
Hi. Thank you for creating this. 

Please create a separate PR for super_trend. @cryptocoinserver has to decide on that. 

As for the description, is it like for the entire trade? (and associated orders of it)
We already have the logger module that is supposed to be used for debugging. Although I don't think I have documented it yet.

**Comment by miguelromero2000** [comment]
I have improved the labels, hidding the test, and adding a tooltip per each label order:
image

**Comment by miguelromero2000** [comment]
Hi @saleh-mir , 

Updated in my last comment all the changes with the current behavior.

_Please create a separate PR for super_trend. @cryptocoinserver has to decide on that._
- Already done, deleted supertrend changes :)

_As for the description, is it like for the entire trade? (and associated orders of it)
We already have the logger module that is supposed to be used for debugging. Although I don't think I have documented it yet._
- I have tested it with the following command:  jesse backtest '2019-03-01' '2021-01-24' --tradingview-debug
- I have seen that the limit in TradingView is about 100 indicators. (50 per script). Since I have sepparated now the tradingview-debug in another strategy file, we can import normal trandingview file and debug file both in our Tradingview panel, and see a total of 100 indicators.
- And yes, it works with the entire trade.

Example of code to use it:
` def should_sell_long(self) -> bool:
    superTrendTuple_4h = self.supertrend_4h
    anchor_trend_ema_100, emaValue_100 = self.anchor_trend_ema_100

    if self.position.is_open == True:  # si tenemos una compra
        if anchor_trend_ema_100 == -1:
            self.description = '- EMA_100 crossed, selling, downtrend\n'
            return True

        if superTrendTuple_4h[2] == 'downtrend':
            self.description = '- supertrend_4h selling, downtrend\n'
            return True

    return False`

**Comment by saleh-mir** [comment]
Thank you. 

This is a bit in conflict with a feature that I'm going to implement later which is to add descriptions per each order. The api would probably be something like:
```
self.buy = [
    (qty1, price1, description1),
    (qty2, price2, description2),
    (qty3, price3, description3),
]
```

Also, we have the `logger.info` that can be used for logging info per trade. I very much appreciate the trouble you went for this tho.
